SpatialGDK/Documentation/content/get-started/gdk-and-starter-project.md

```diff
@@ -0,0 +1,63 @@
+    1. In File Explorer, navigate to the root directory of the Unreal GDK Starter Project repository, and then to the **Game** directory within it.
+    2. Right-click **StarterProject.uproject** and select **Switch Unreal Engine version**.
+    3. Select the path to the Unreal Engine fork you cloned earlier.
+3. In the same directory, open **StarterProject.sln** with Visual Studio.
+4. In the Solution Explorer window, right-click on **StarterProject** and select **Build**.
+5. Open **StarterProject.uproject** in the Unreal Editor and click [`Schema` (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/glossary) to generate schema and then [`Snapshot` (SpatialOS documentation)]((https://docs.improbable.io/reference/latest/shared/glossary)) to generate a snapshot.
+
+### Deploy the Starter Project locally
+
+1. In the Unreal Editor, on the SpatialOS GDK toolbar, click **Launch**. Wait until you see the output `SpatialOS ready. Access the inspector at http://localhost:21000/inspector`.
+2. On the Unreal Editor toolbar, open the **Play** drop-down menu.
+3. Under **Multiplayer Options**, enter the number of players as **2** and check the box next to **Run Dedicated Server**. Then, under Modes, select **New Editor Window (PIE)**.
+4. On the toolbar, click **Play** to run the game.
+5. When you're done, click **Stop** to stop the client [worker](https://docs.improbable.io/reference/latest/shared/glossary), and click the **Stop** button in the SpatialOS section of the toolbar to stop your local SpatialOS deployment.
+
+### Deploy the Starter Project in the cloud
+
+To run a cloud deployment, you need to prepare your server-worker and client-worker [assemblies](https://docs.improbable.io/reference/latest/shared/glossary), and upload them to the cloud.
+
+> Building the assemblies can take a while - we recommend installing [IncrediBuild](https://www.incredibuild.com/), [FastBuild](http://www.fastbuild.org/), or another build distributor.
+
+1. Change the name of the project
+    1. In File Explorer, navigate to the root directory of the Starter Project repository, then to **`\spatial`**, and open the `spatialos.json` file in an editor of your choice.
+    2. Change the `name` field to the name of your project. You can find this in the [Console](https://console.improbable.io). It’ll be something like `beta_someword_anotherword_000`.
+2. In a terminal window, navigate to the root directory of the Starter Project repository.
+3. Build a server-worker assembly: `Game\Plugins\UnrealGDK\SpatialGDK\Build\Scripts\BuildWorker.bat StarterProjectServer Linux Development StarterProject.uproject`
+4. Build a client-worker assembly: `Game\Plugins\UnrealGDK\SpatialGDK\Build\Scripts\BuildWorker.bat StarterProject Win64 Development StarterProject.uproject`
+5. Navigate to `StarterProject/spatial`.
+6. Upload the assemblies to the cloud, specifying an assembly name (this covers both assemblies): `spatial cloud upload <assembly_name>`
```

Comment:
"this covers both assemblies" -> "this includes both client-worker and server-worker assemblies"

Reply:
thanks updated

Reply:
thanks updated